app/controllers/publish/courses/funding_type_controller.rb

```diff
@@ -38,7 +38,7 @@ def update
       def funding_type_params
         return {} if params[:publish_course_funding_form].blank?
 
-        params.require(:publish_course_funding_form).permit(:funding_type, :previous_tda_course)
+        params.require(:publish_course_funding_form).permit(:funding_type, :funding, :previous_tda_course)
```

Comment:
How do you feel about putting a comment above this to the effect that when we get rid of `FeatureService.enabled?(:db_backed_funding_type)` we need to remove `:funding_type` from this line? It feels like something we'll forget otherwise. But if we are looking everywhere for `FeatureService.enabled?(:db_backed_funding_type)` and we find it in this comment, we'll remember. 

I'm not sure how people feel about comments / TODOs in the code. So if it's not the done thing, happy for this to be ignored. 

Basically, I can't find anything wrong with this and I'm just making a comment so you can tell I've done a thorough review.

Reply:
Haha, thats great. Thank you 🙏🏼 

I'm usually against comments in code, I think the code should be self explanatory but in this case we could easily miss it since the feature flag isn't directly referenced so happy to add the comment.

I'm planning to keep the feature flag in long enough to ensure that everything is working as expected but short enough that it's still fresh in my mind. So the comment will be removed soon anyways.

https://github.com/DFE-Digital/publish-teacher-training/pull/4467/commits/aeaaaac445b86c74e6d5775a9cac5eaadfff2d9a